shop/src/main/java/ro/msg/learning/shop/OnlineShopApplication.java

```diff
@@ -0,0 +1,18 @@
+package ro.msg.learning.shop;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan("ro.msg.learning.shop.model")
```

Comment:
@SpringBootApplication already contains the @ComponentScan annotation with the default value, which means current package and all sub-packages (so all the packages mentioned by you here). So there is no need to add the @ComponentScan